pkgs/development/libraries/volume-key/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchgit, fetchpatch, autoreconfHook, pkgconfig, gettext, python2
```

Comment:
python2: this is inconsistent with libblockdev above, which uses python3 and depends on volume_key. Please use python3 here if supported.

Reply:
python3 is not supported according to https://pagure.io/volume_key/pull-request/12